src/styles/viewer.styl

```diff
@@ -50,6 +51,7 @@
     position        relative
     display         flex
     flex-grow       1
+    max-height      100%
```

Comment:
Well, don't. We'll fix this later if we see it bugging. Fixing a bug that doesn't exist yet is the best way to add a new bug. :)